server.go

```diff
@@ -8,20 +8,26 @@ import (
 	"net/http"
 	"bytes"
 	"strings"
-
-	"github.com/clarifai/clarifai-go"
 	"github.com/gin-gonic/gin"
+	"os"
+	"net/url"
 )
 
 const (
 	clarifaiApi = "https://api.clarifai.com/v2/models/aaa03c23b3724a16a56b629203edc62c/outputs"
+	clarifyAuth = "https://api.clarifai.com/v1/token"
```

Comment:
Maybe we should use /v2/token in order to keep consistent. We probably don't want to use two versions of API